### PR TITLE
Refactor commons and logic test cases

### PR DESCRIPTION
**Pull request by bryanhce**
Some imported classes were not renamed as they are under other packages.

### CODE REVIEW

File: src/test/java/seedu/guest/logic/commands/AddCommandIntegrationTest.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ public void setUp() {
     }
 
     @Test
-    public void execute_newPerson_success() {
+    public void execute_newGuest_success() {
         Guest validGuest = new PersonBuilder().build();
```
Comment (kangzongxian) [inline src/test/java/seedu/guest/logic/commands/AddCommandIntegrationTest.java]: Should this be GuestBuilder.build(), or should it not be refactored? Found in seedu.address.testutil.PersonBuilder

File: src/test/java/seedu/guest/logic/commands/ClearCommandTest.java [hunk 2]
```diff
@@ -13,15 +13,15 @@
...
         Model model = new ModelManager();
         Model expectedModel = new ModelManager();
 
         assertCommandSuccess(new ClearCommand(), model, ClearCommand.MESSAGE_SUCCESS, expectedModel);
     }
 
     @Test
-    public void execute_nonEmptyAddressBook_success() {
+    public void execute_nonEmptyGuestBook_success() {
         Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```
Comment (kangzongxian) [inline src/test/java/seedu/guest/logic/commands/ClearCommandTest.java:25]: Should this be getTypicalGuestBook() as well? Found in seedu.address.testutil.TypicalPersons

File: src/test/java/seedu/guest/logic/commands/DeleteCommandTest.java [hunk 3]
```diff
@@ -49,7 +49,7 @@ public void execute_invalidIndexUnfilteredList_throwsCommandException() {
 
     @Test
     public void execute_validIndexFilteredList_success() {
-        showPersonAtIndex(model, INDEX_FIRST_PERSON);
+        showGuestAtIndex(model, INDEX_FIRST_PERSON);
```
Comment (kangzongxian) [inline src/test/java/seedu/guest/logic/commands/DeleteCommandTest.java:52]: Should this be INDEX_FIRST_GUEST as well? Found in seedu.address.testutil.TypicalIndexes

File: src/test/java/seedu/guest/logic/commands/AddCommandIntegrationTest.java (hunk 1 above)
Comment (bryanhce) [reply src/test/java/seedu/guest/logic/commands/AddCommandIntegrationTest.java]: This should be refactored under Zong Xian's (your) part as it is this under the testutil package.

File: src/test/java/seedu/guest/logic/commands/ClearCommandTest.java (hunk 2 above)
Comment (bryanhce) [reply src/test/java/seedu/guest/logic/commands/ClearCommandTest.java:25]: This should be refactored under Zong Xian's (your) part as it is this under the testutil package.

File: src/test/java/seedu/guest/logic/commands/DeleteCommandTest.java (hunk 3 above)
Comment (bryanhce) [reply src/test/java/seedu/guest/logic/commands/DeleteCommandTest.java:52]: This should be refactored under Zong Xian's (your) part as it is this under the testutil package.

**Pull request merged**